docs/PROTOCOL.rst

```diff
@@ -532,7 +532,7 @@ blockchain.transaction.get
 
 Return a raw transaction.
 
-  blockchain.transaction.get(**tx_hash**, **height**)
+  blockchain.transaction.get(**tx_hash**, **height**, **verbose** = [False])
```

Comment:
No, this 1.0 API with height isn't changing.

You need to make a copy under "Version 1.2" and remove the height parameter (see the notes under Version 1.1).   I know the layout of the protocol information is awkward; I will clean that up later.